packages/desktop-client/src/components/sidebar/Account.tsx

```diff
@@ -149,3 +153,88 @@ export function Account({
     </View>
   );
 }
+
+export function Account({
+  name,
+  account,
+  connected,
+  pending = false,
+  failed,
+  updated,
+  to,
+  query,
+  style,
+  outerStyle,
+  onDragChange,
+  onDrop,
+}: AccountProps) {
+  const accountNote = useNotes(`account-${account?.id}`);
+  const needsTooltip = !!account?.id;
+
+  return needsTooltip ? (
+    <Tooltip
+      content={
+        <View
+          style={{
+            padding: 10,
+          }}
+        >
+          <Text
+            style={{
+              fontWeight: 'bold',
+              borderBottom: accountNote ? `1px solid ${theme.tableBorder}` : 0,
+              marginBottom: accountNote ? '0.5rem' : 0,
+            }}
+          >
+            {name}
+          </Text>
+          {accountNote && (
+            <Notes
+              getStyle={() => ({
+                padding: 0,
+              })}
+              notes={accountNote}
+            />
+          )}
+        </View>
+      }
+      className={`${css({
+        borderRadius: '0px 5px 5px 0px ! important',
+      })}`}
+      placement="right top"
+      triggerProps={{
+        delay: 1000,
+      }}
+    >
+      {AccountRow(
+        account,
+        outerStyle,
+        onDragChange,
+        onDrop,
+        to,
+        style,
+        updated,
+        pending,
+        failed,
+        connected,
+        name,
+        query,
+      )}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏two suggestions in one comment

1. could you assign the `AccountRow` to a variable and then reuse that? I.e. `const myAccountRow = <AccountRow {allTheProps} />;` &&& `<Tooltip>{myAccountRow}</Tooltip>` - that way we reduce the duplication
2. would you mind changing the code structure to return early?

I.e. prefer this..

```
if (needsTooltip) {
   return <Tooltip>{myAccountStuff}</Tooltip>;
}

return myAccountStuff;
```

over this..

```
return needsTooltip ? <Tooltip>{myAccountStuff}</Tooltip> : myAccountStuff;
```

especially if the code block is VERYYY long.